client/src/components/Post/Post.jsx

```diff
@@ -6,19 +6,21 @@ import useStyles from './postStyles'
 import PostAuthor from './PostAuthor/PostAuthor'
 import PostLikePanel from './PostLikePanel/PostLikePanel'
 import PostComments from './PostComments/PostComments'
+import CreateComment from './CreateComment/CreateComent'
 
 const Post = ({ post }) => {
   const classes = useStyles()
 
-  const { id, author, owner, date, message, image, comments } = post
+  const { id, author, owner, date, message, image, likes, comments } = post
 
   return (
     <Paper key={id} className={classes.post}>
       <PostAuthor author={author} owner={owner} date={date}/>
       <img src={image} className={classes.postImg} alt='Post' />
       <p>{message}</p>
-      <PostLikePanel id={id} post={post} comments={comments} />
+      <PostLikePanel id={id} likes={likes} comments={comments} />
```

Comment:
Let's rename id to postId, cause it's confusing inside PostLikePanel

Reply:
Ok